plugins/carlabase/carla.cpp

```diff
@@ -150,15 +151,22 @@ CarlaInstrument::CarlaInstrument(InstrumentTrack* const instrumentTrack, const D
     fHost.uiName      = NULL;
     fHost.uiParentId  = 0;
 
-    // figure out prefix from dll filename
+    // carla/resources contains PyQt scripts required for launch
     QString dllName(carla_get_library_filename());
-
+    QString resourcesPath = QString();
```

Comment:
You can leave out the ` = QString()` here.
C++ classes initialize to their default if not provided a value.